locale/foreman.pot

```diff
@@ -15198,28 +15198,28 @@ msgid "Filter Host..."
 msgstr ""
 
 #:
-#: ../webpack/assets/javascripts/react_app/components/Editor/components/EditorNavbar.js:77
+#: ../webpack/assets/javascripts/react_app/components/Editor/components/EditorToolbar.js:77
```

Comment:
Usually we don't update this file. If we do, we do it as part of the string extraction process, which is automated.

Reply:
> Usually we don't update this file. If we do, we do it as part of the string extraction process, which is automated.

OK, I'm sorry if I was wrong to do so 😅 
I think I made these changes automatically when I updated the name of the component (and by that updated all the imports) or when I ran this command `npm test -- -u` in order to update the snapshots
I have changed the name of the component 'EditorNavbar' to 'EditorToolbar' as part of this refactor so it would better fit it's implementation
Would you recommend keeping the name as it was before and avoid this change? 
How would I run the string extraction process (if it is not part of the automated checks that run when I push a PR) ?
Could you explain what is the purpose of this file ? (`foreman.pot`)
Thank you @ekohl 🙏🏼

Reply:
Thank you @MariaAga for your help ! 
This change was removed 👍🏼